### PR TITLE
fix(RHINENG-9387): update page titles

### PR DESCRIPTION
**Pull request by mkholjuraev**
## PR Title :boom:

This updates the page titles according to this [ticket](https://issues.redhat.com/browse/RHINENG-9387) and [sheet](https://docs.google.com/spreadsheets/d/1Uhwn6i9B383xUbnRQ2IWY3EZ6dihc-UfkHxnR_c7zXI/edit#gid=1001620868)

Suggested formats: 

1. Fixes/Refs #RHIROS-XXX - Title
2. RHIROS-XXX Title 

Feel free to remove this section from PR description once done.

## Why do we need this change? :thought_balloon:

Please include the __context of this change__ here.

## Documentation requires update? :memo:

- [ ] Yes
- [ ] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [ ] New Feature
- [ ] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

**Comment by r14chandra** [comment]
\retest

**Comment by gkarat** [comment]
@mkholjuraev, @PreetiW, there is a linter failing on both ci.int and Travis. The issue is really weird and seems to have started happening with this PR. Though I don't really understand where this `range` comes from:
```
TypeError: Cannot read properties of null (reading 'range')
Occurred while linting /home/travis/build/RedHatInsights/ros-frontend/src/Routes/RosSystemDetail/RosSystemDetail.js:1
```

**Comment by mkholjuraev** [comment]
@PreetiW @gkarat This maybe due to the old unsupported version of babel-eslint package. @PreetiW I am wondering if you are going to migrate away from this package?

**Comment by gkarat** [comment]
Can we still merge the PR without waiting for this upgrade? The PR changes are minor and only touch the page titles.

File: src/Routes/RosSystemDetail/RosSystemDetail.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ class RosSystemDetail extends React.Component {
         const chrome = this.props.chrome;
         const displayName = this.props.rosSystemInfo.display_name;
         if (displayName && displayName !== document.title) {
-            chrome?.updateDocumentTitle(displayName);
+            chrome?.updateDocumentTitle(`${displayName} - Resource Optimization - Business`);
```
Comment (PreetiW) [inline src/Routes/RosSystemDetail/RosSystemDetail.js:54]: @mkholjuraev for now can you please put the title in a variable and use it? for eg:
`const detailPageTitle = `${displayName} - Resource Optimization - Business`;
 chrome?.updateDocumentTitle(detailPageTitle);`

I checked the lint is not complaining with this. I will look into the lint issue later around why it gives error with above code. 
Thank you!

**Review by PreetiW — CHANGES_REQUESTED** [review]
Thank you for the PR @mkholjuraev. I have requested a minor change, let me know once its done and I will go ahead and merge the PR. 🙌🏻

**Review by PreetiW — APPROVED** [review]
@mkholjuraev Thank you for making the changes. Merging it 💯

**Pull request merged**